build.cake

```diff
@@ -58,11 +58,11 @@ Task("Test")
     int i = 0;
     var testSettings = new DotNetCoreTestSettings
     {
-        Configuration = configuration,
+        Configuration = "Debug",
```

Comment:
Is this because of the open Snapshooter issue?

Reply:
Yes. We discussed in the Team and figured that it doesn't make a lot of sense to Test in Release Mode if you don't constantly use the pre-processor directives.

Reply:
Ok.
Well I think they should at least behave the same way.